### PR TITLE
Move to external test matrix

### PR DESCRIPTION
**Pull request by CAMOBAP**
Related: https://github.com/fontist/fontist/issues/308

### CODE REVIEW

**Comment by ronaldtse** [comment]
Ruby 3.2 failing but we can fix them later.

**Comment by ronaldtse** [comment]
@CAMOBAP however I don't know if the failing specs will block the current GHA release flow...

**Comment by CAMOBAP** [comment]
@ronaldtse they will not, I'm merging and releasing this PR

File: .github/workflows/metanorma.yml
Comment (maxirmx) [inline .github/workflows/metanorma.yml:1]: Although it is too late but it looks like external test matrix is very unfriendly for any development _outside of_ or _ahead of_ metanorma project.

Comment (CAMOBAP) [reply .github/workflows/metanorma.yml:1]: Any suggestion on how this can be improved? Just to understand what means by "unfriendly"

**Pull request merged**